runelite-client/src/main/java/net/runelite/client/plugins/barrows/CryptMonsters.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2018, Seth <Sethtroll3@gmail.com>
```

Comment:
You aren't Seth, change this to 2020 and use your name and contact email/website